tests/r7rs/r7rs-tests.scm

```diff
@@ -537,7 +546,8 @@
 
 (test 'ok (let ((=> #f)) (cond (#t => 'ok))))
 
-(let ()
+; TODO(higepon): multiple definitions of identifier.
+#;(let ()
   (define x 1)
   (let-syntax ()
```

Comment:
R7RS `let-syntax` allows to have definitions. To implement it using R6RS, for example, it can be seen as it was surrounded with `(let () ...)`.

https://github.com/okuoku/yuni/commit/53996c38bfb2c22f0ce15bf7a9fd6ff835b049bf#diff-f2d93f25147380c152d5e92117ff78c85d505e1e974ed02d805ee07b614bef78R105-R108

R7RS says it might have "one-or-more" definitions but it is known as errata: https://github.com/johnwcowan/r7rs-work/blob/994cc5726ae71adf4eb09ebcd95df2d9a607f5d5/R7RSSmallErrata.md#L95 obviously it is "zero-or-more".

Reply:
Thank you. Noted in the source code.